src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs/logs.component.ts

```diff
@@ -27,6 +42,75 @@ export class LogsComponent implements OnInit, OnDestroy {
   getInfo() {
     this.logsService.getLogs().subscribe((data: any) => {
       this.contentData = data;
+      this.filterLogs();
+    });
+  }
+
+  transformDateType(): any {
+    let yearMonthDay: string;
+    let sTime: number, eTime: number;
+
+    if (this.selectedDate) {
+      yearMonthDay = this.selectedDate.toISOString().slice(0, 10);
+    } else {
+      yearMonthDay = null;
+    }
+
+    const sHour = this.startTime ? this.startTime.getUTCHours() : 0;
```

Comment:
```suggestion
    const sHour = this.startTime ? this.startTime.getHours() : 0;
```


Do not use UTC functions here, otherwise it happens that if you enter 11 (in hours) you will get 10 as a result here (currently i've CET which means UTC+1). Another reason why you shouldn't use time zone related functions is because you're not doing any time zone adaption in ``calcLogStamp()``.

With the current code you'll get a result that does not match the entered filter parameters. The UI still displays the log entries that happen at 10:xx o'clock.

![auswahl_003](https://user-images.githubusercontent.com/1897962/53877856-92e76680-400a-11e9-9341-50cbd541e2a3.png)